test/flows/calculate_statutory_sick_pay_test.rb

```diff
@@ -0,0 +1,702 @@
+      end
+    end
+  end
+
+  context "question: days_covered_by_earnings?" do
+    setup do
+      testing_node :days_covered_by_earnings?
+      add_responses is_your_employee_getting?: "statutory_adoption_pay",
+                    coronavirus_related?: "no",
+                    employee_tell_within_limit?: "yes",
+                    employee_work_different_days?: "no",
+                    first_sick_day?: "2020-01-01",
+                    last_sick_day?: "2020-01-05",
+                    has_linked_sickness?: "yes",
+                    linked_sickness_start_date?: "2019-12-01",
+                    linked_sickness_end_date?: "2019-12-30",
+                    paid_at_least_8_weeks?: "eight_weeks_less",
+                    total_earnings_before_sick_period?: "700000"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of usual_work_days? for any valid response" do
+        assert_next_node :usual_work_days?, for_response: "5"
+      end
+    end
+  end
+
+  context "question: usual_work_days?" do
+    setup do
+      testing_node :usual_work_days?
+      add_responses is_your_employee_getting?: "statutory_adoption_pay",
+                    coronavirus_related?: "no",
+                    employee_tell_within_limit?: "yes",
+                    employee_work_different_days?: "no",
+                    first_sick_day?: "2020-01-01",
+                    last_sick_day?: "2020-01-05",
+                    has_linked_sickness?: "yes",
+                    linked_sickness_start_date?: "2019-12-01",
+                    linked_sickness_end_date?: "2019-12-30",
+                    paid_at_least_8_weeks?: "eight_weeks_less",
+                    total_earnings_before_sick_period?: "700000",
+                    days_covered_by_earnings?: "5"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have an outcome of not_earned_enough for any response if employee_average_weekly_earnings is less than lower_earning_limit_rate" do
+        add_responses linked_sickness_start_date?: "2019-12-01",
+                      linked_sickness_end_date?: "2019-12-30",
+                      paid_at_least_8_weeks?: "eight_weeks_less",
+                      total_earnings_before_sick_period?: "70",
+                      days_covered_by_earnings?: "5"
+
+        assert_next_node :not_earned_enough, for_response: "4"
+      end
+
+      should "have an outcome of maximum_entitlement_reached for any response if too many previous sick days" do
+        add_responses linked_sickness_start_date?: "2018-12-01",
+                      linked_sickness_end_date?: "2019-12-30",
+                      paid_at_least_8_weeks?: "eight_weeks_less",
+                      total_earnings_before_sick_period?: "70000",
+                      days_covered_by_earnings?: "5"
```

Comment:
Same here. I don't think these values need to be set again as they are the same as the defaults in the `setup`.